src/lib/form-field/_form-field-legacy-theme.scss

```diff
@@ -56,6 +56,16 @@ $mat-form-field-legacy-dedupe: 0;
   $mat-form-field-legacy-dedupe: $mat-form-field-legacy-dedupe + 0.00001 !global;
 }
 
+// Same as mixin above, but omits the translateZ for printing purposes.
+@mixin _mat-form-field-legacy-label-floating-print($font-scale, $infix-padding, $infix-margin-top) {
+  // This results in a small jitter after the label floats on Firefox, which the
+  // translateZ fixes.
+  transform: translateY(-$infix-margin-top - $infix-padding + $mat-form-field-legacy-dedupe) scale($font-scale);
```

Comment:
It looks like there's a lint error because this line is too long, can you put the `scale($font-scale)` part on the next line?